docs/sources/enterprise/license-restrictions.md

```diff
@@ -0,0 +1,44 @@
++++
+title = "License restrictions"
+description = "Grafana Enterprise license restrictions"
+keywords = ["grafana", "licensing", "enterprise"]
+weight = 8
++++
+
+# License restrictions
+
+Enterprise licenses are limited by the number of active users, a license expiration date, and the URL of the Grafana instance.
+
+**User limits**
```

Comment:
All the bold text pretending to be headings should actually be headings.  That way they appear in the side menu.